new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Installer.java

```diff
@@ -35,25 +37,25 @@ public boolean download() {
     boolean downloadResult = true;
     boolean unzipResult = true;
     if (params.verify()
-        && new File(downloadDir, params.fileName()).exists()) {
+        && new File(DOWNLOAD_DIR, params.fileName()).exists()) {
       logger.info("File " + params.fileName() + " already exists.");
     } else {
       checkDirectory(WORK_DIR);
       downloadResult = executeAndVerify(buildDownloadCommand(), params.direct());
     }
-    if (!(new File(downloadDir + "../imagetool/imagetool.sh").exists()) 
+    if (!(new File(IMAGE_TOOL).exists()) 
```

Comment:
i am not sure what this is meant to do.
are you trying to say - if the file DOES exist, then unzip it?
or if it DOES NOT exist, and you asked me to unzip, then unzip it?
seems we could write this with code that is longer, but much clearer

Reply:
This is try not to unzip it more than once.  I'll make it clearer.

Reply:
Modified the code.